src/timepicker/docs/readme.md

```diff
@@ -50,3 +50,7 @@ All settings can be provided as attributes in the `<uib-timepicker>` or globally
  * `max`
     _(Defaults: undefined)_ :
      Maximum time a user can select
+
+ * `timepicker-tabindex`
```

Comment:
couldn't we use `tabindex` here? And remove it from the timepicker host element if needed?